src/org.xtuml.bp.ui.properties/arc/BuildPropertySource.arc

```diff
@@ -757,7 +757,8 @@ val = val${count};
               Attribute_c base_attr = Attribute_c.getOneO_ATTROnR106(BaseAttribute_c.getOneO_BATTROnR113(rattr));
               if ( (  (base_attr == null)) )
               {
-                return "same_as<Base_Attribute>";
```

Comment:
It is hard to understand archetype changes without resulting source.  I do not understand this change and do not see a description of it in the note. Please explain. I prefer short explanation in the note.